src/ResourceManager/Sql/Commands.Sql/Elastic Pools/Services/AzureSqlElasticPoolCommunicator.cs

```diff
@@ -121,6 +121,14 @@ public Management.Sql.Models.ElasticPool CreateOrUpdate(string resourceGroupName
                 createOrUpdateResponse, null, CancellationToken.None).Result.Body;
         }
 
+        /// <summary>
+        /// Updates an Elastic Pool using Patch
+        /// </summary>
+        public Management.Sql.Models.ElasticPool CreateOrUpdate(string resourceGroupName, string serverName, string elasticPoolName, Management.Sql.Models.ElasticPoolUpdate parameters)
+        {
+            return  GetCurrentSqlClient().ElasticPools.UpdateWithHttpMessagesAsync(resourceGroupName, serverName, elasticPoolName, parameters).Result.Body;
```

Comment:
Instead of `UpdateWithHttpMessagesAsync.Result.Body`, you can just call `Update`